presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -166,19 +166,70 @@ queries.
     * Checks the nullity of control sum and test sum.
     * If either control mean or test mean very close 0, checks if both are close to 0.
     * Checks the relative error between control sum and test sum.
+* **VARCHAR Columns**
+    * For ``VARCHAR`` columns, a simple checksum column is generated for verification using :func:`!checksum`.
+    * If ``validate-string-as-double`` is set, 6 additional columns below are generated. If ``NULL`` counts
+      are equal before and after casting all values to ``DOUBLE``, apply the aforementioned floating point validation.
+      Otherwise, check if the simple checksum matches.
+        * Count of the ``NULL`` values
+        * Count of the ``NULL`` values after all values are casted to ``DOUBLE``
+        * After casting all values to ``DOUBLE``
+            * Sum of the finite values
+            * ``NAN`` count of the values
+            * Positive infinity count of the values
+            * Negative infinity count of the values
 * **Array Columns**
-    * 2 columns are generated for verification:
-       * Sum of the cardinality
-       * Array checksum
-    * For an array column ``arr`` of type ``array(E)``:
-       * If ``E`` is not orderable, array checksum is ``checksum(arr)``.
-       * If ``E`` is orderable, array checksum ``coalesce(checksum(try(array_sort(arr))), checksum(arr))``.
+    * For an array column ``arr`` of type ``array(E)``, 3 columns are generated for verification:
+        * Sum of the cardinality
+        * Checksum of the cardinality
+        * Array checksum
```

Comment:
```suggestion
        Array checksum
```
Suggest not formatting this line as an item in the list, to help the reader. The two sub-items formatting is good, as your description clearly tells the reader the next two are an either/or.

Reply:
Need to keep this as one item in the list. "Array checksum" is one of the three columns generated.